src/bindings/js/node/src/core_wrap.cpp

```diff
@@ -85,11 +85,13 @@ Napi::Value CoreWrap::read_model_sync(const Napi::CallbackInfo& info) {
             model = _core.read_model(model_str, weight_tensor);
         } else if (ov::js::validate<Napi::String>(info, allowed_signatures)) {
             model = _core.read_model(info[0].ToString());
+        } else if (ov::js::validate<Napi::String, TensorWrap>(info, allowed_signatures)) {
+            model = _core.read_model(info[0].ToString(), cast_to_tensor(info, 1));
         } else {
             OPENVINO_THROW("'readModelSync'", ov::js::get_parameters_error_msg(info, allowed_signatures));
         }
```

Comment:
Probably, it makes sense to use `ReadModelArgs` here.
(I prefer see validation parameters here, but `ReadModelArgs` reduce code in async case. Here could be an exception to do not duplicate logic in two types of readModel methods)

Reply:
`ReadModelArgs` and other structs are marked in *146325* to be refactored